app/src/main/java/protect/card_locker/importexport/MultiFormatImporter.java

```diff
@@ -59,6 +64,7 @@ public static ImportExportResult importData(Context context, SQLiteDatabase data
                 error = e.toString();
             } finally {
                 database.endTransaction();
+                inputFile.delete();
```

Comment:
not checking the return value here is what spotbugs complains about. not sure how best to handle.

Reply:
I know spotBugs should allow you to ignore checks but I've always found that very finicky and can never find the syntax online. I think it might be good to just check the result and if it didn't delete successfully `Log.w` it because it is a situation that shouldn't happen? Although, as I understand the code from a quick glance, it should cause the inputFile to be overwritten next import attempt so we may want to write down in the log message that this should not be a problem.

Reply:
I was already thinking adding a `Log.w()` or `Log.d()` made the most sense. As you say it shouldn't happen, but it's a cache file and it doesn't make sense to me to fail the import if removing it fails, which is why I didn't check the result. Will add that.